pipelines/MetricsMiscWithFakes.yaml

```diff
@@ -0,0 +1,44 @@
+# Miscellaneous metrics for Alert Production
+# In the future, these might be placed in task-specific pipelines (for debugging)
+# or grouped by their datasets (to optimize expensive Butler reads)
+
+description: Miscelaneous AP Pipeline metrics
+tasks:
+    numNewDiaObjects:
+        class: lsst.ap.association.metrics.NumberNewDiaObjectsMetricTask
+        config:
+            connections.labelName: diaPipe  # partial name of metadata dataset
+    numUnassociatedDiaObjects:
+        class: lsst.ap.association.metrics.NumberUnassociatedDiaObjectsMetricTask
+        config:
+            connections.labelName: diaPipe
+    fracUpdatedDiaObjects:
+        class: lsst.ap.association.metrics.FractionUpdatedDiaObjectsMetricTask
+        config:
+            connections.labelName: diaPipe
+    totalUnassociatedDiaObjects:
+        class: lsst.ap.association.metrics.TotalUnassociatedDiaObjectsMetricTask
+    numSciSources:
+        class: lsst.ip.diffim.metrics.NumberSciSourcesMetricTask
```

Comment:
Why not replace these with an import? Or (given my advice to import `ApVerify.yaml` above) why not create a pipeline file that only has the fakes-specific metrics, then import it and `MetricsMisc.yaml` separately? As it stands, the files need to be kept in sync if somebody creates a new metric.